types/zxcvbn/index.d.ts

```diff
@@ -10,54 +11,71 @@ export = zxcvbn;
 declare function zxcvbn(password: string, userInputs?: string[]): zxcvbn.ZXCVBNResult;
 
 declare namespace zxcvbn {
+    type ZXCVBNFeedbackWarnings = '' | 'Straight rows of keys are easy to guess' | 'Short keyboard patterns are easy to guess' |
```

Comment:
I don't think these should be part of a string enum, they are human readable strings not meant for analytic consumption.

Reply:
I concur with @SimonSchick, there doesn't seem to be a good reason for these to be individually typed.